src/bro/connector/slack.py

```diff
@@ -130,6 +131,6 @@ def send(self, message: Message, via: Channel) -> None:
     logs.setup(log_file=LOG_FILE, db_file=DB_FILE)
     _logger.setLevel(logging.INFO)
 
-    connector = SlackConnector(os.environ["SLACK_BOT_TOKEN"], os.environ["SLACK_APP_TOKEN"])
+    connector = SlackConnector(os.environ["SLACK_BOT_TOKEN"], os.environ["SLACK_APP_TOKEN"], os.environ["BRO_USER_ID"])
```

Comment:
It is best to use keyword arguments. The constructor definition should also be updated to require [only keyword arguments](https://peps.python.org/pep-3102/).